instrumentation/vertx-core-4.5.0/build.gradle

```diff
@@ -0,0 +1,22 @@
+dependencies {
```

Comment:
This module is basically a copy of the 4.3.2 module.
The difference is a class that was renamed, and implementing a new method that was added to an interface.